osg_configure/modules/utilities.py

```diff
@@ -646,26 +639,4 @@ def reconfig_service(service, reconfig_cmd):
         logger.info("Reconfigure successful")
         return True
 
-    return False
-
-
-
-class NullLogger(logging.Logger):
-    """A dummy Logger where the logging functions ignore all parameters
-    passed to it.  They are static methods so you don't need to instantiate
-    it.  Useful as a default value for functions which take a logger as an
-    argument.
-
-    """
-    @staticmethod
-    def info(*args, **kwargs): pass
-    @staticmethod
-    def warning(*args, **kwargs): pass
-    @staticmethod
-    def error(*args, **kwargs): pass
-    @staticmethod
-    def critical(*args, **kwargs): pass
-    @staticmethod
-    def debug(*args, **kwargs): pass
-    @staticmethod
-    def log(*args, **kwargs): pass
+    return False
```

Comment:
```suggestion
    return False
    
```

Github doesn't let me suggest _just_ adding a newline at eof, which is my intent here ... but an extra newline is ok by me also.